pkg/models/monitoring/sort_page.go

```diff
@@ -120,8 +120,10 @@ func (raw *Metrics) Sort(target, order, identifier string) *Metrics {
 				// Record ordinals in the final result
 				v, ok := mv.Metadata[identifier]
 				if ok && v != "" {
-					resourceOrdinal[v] = ordinal
-					ordinal++
+					if _, ok1 := resourceOrdinal[v]; !ok1 {
```

Comment:
It's ok to have a local variable named `ok`, `ok1` is wired. Check https://go.dev/play/p/ISoojvwfgu_S